andela/codango
# Create new home page Slider

#### What does this PR do?
Create a Landing page in isolation
#### Description of Task to be completed?
Designing of the slider for the new Homepage along with the navigation.
#### How should this be manually tested?
Visit [codango-staging-pr-149.herokuapp.com](https://codango-staging-pr-149.herokuapp.com) click on the "new" menu on the navbar to see the new landing page

#### What are the relevant pivotal tracker stories?
#128843781- Create Home Page

#### Screenshots
![screen shot 2016-08-24 at 12 44 54 pm](https://cloud.githubusercontent.com/assets/17270426/17929654/c46ad514-69f9-11e6-9297-bca7f6dcf0a9.png)
